solc/CommandLineInterface.cpp

```diff
@@ -131,26 +131,6 @@ static string const g_strSrcMapRuntime = "srcmap-runtime";
 static string const g_strStorageLayout = "storage-layout";
 static string const g_strVersion = "version";
 
-static bool needsHumanTargetedStdout(CommandLineOptions const& _options)
```

Comment:
Why did you remove this function? (this is likely why the tests are failing)

Reply:
That function was only called on line 1194 to check whether to print the contract header. The issue specified the header be always printed so the check was removed. The function was used no where else and as such would cause a compiler error when left it.

Reply:
For instance, if `_options.output.dir.empty() == false` (see line 138), it wouldn't have printed the content on line 1145, whereas now it does. I will admit that I haven't read the original issue in full (just the original description), so I'll take a closer look tomorrow :)

Reply:
OK, so I took a look, and you definitely don't want to delete `needHumanTargetStdout` - the issue itself has nothing to do with this; removing it essentially fails all of the `cmdlineTests` as you're adding another line to pretty much all of them.
The only thing that's really required in this PR is to make sure that if there are no contracts present, to output such a message (which you've already done); this also needs to be covered by tests as well.